File: Data Structures & Algorithms/is-anagram/submission-0.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length() != t.length()){
            return false;
        }
        int [] count = new int[26];

        for(char c : s.toCharArray()){
            int index = c - 'a';
            count[index]++;
        }
        
        for(char c: t.toCharArray()){
            int index = c - 'a';
            count[index]--;
        }

        for(int i = 0; i < count.length; i++){
            if(count[i] != 0){
                return false;
            }
        }
        return true;
    }
}

File: Data Structures & Algorithms/daily-temperatures/submission-0.py
class Solution:
    def dailyTemperatures(self, temperatures: List[int]) -> List[int]:
        result = []
        for i in range(len(temperatures)):
            j = i + 1
            count = 0
            found = False
            while(j <= len(temperatures) - 1):
                if(temperatures[j] > temperatures[i]):
                    found = True
                    count += 1
                    result.append(count)
                    break
                else:
                    j +=1
                    count +=1
            if(found == False):
                result.append(0)
        return result
